### DISCUSSION TITLE
Thanks for making my life easier

### DISCUSSION THREAD

**Discussion by Alice1o4** [mdbootstrap/TW-Elements#1652] [Kind words & general feedback]
Thanks for creating this for free.

I hope you release the figma files for the Minimal design system soon, and that they will be compatible with Tailwind.

I didn't like your design at first - I thought it's too empty and "cold", but now I understand your approach and I think your docs are one of the easiest to use and read.

**Comment by ascensus-mdb** [comment]
Thank you for such kind words, it motivates us to work harder on this project!

And we are glad to hear you found Tailwind Elements useful 😉

Figma files for Material Minimal are already in progress and we hope they will be ready soon!
